.github/labels.yml

```diff
@@ -0,0 +1,42 @@
+# Add 'source' label to any change to src files within the source dir EXCEPT for the docs sub-folder
+# source:
+# - any: ['src/**/*', '!src/docs/*']
+
+
+🛤️ Path:
+- 'src/Mod/Path/**/*'
+
+🏛 Arch:
+- 'src/Mod/Arch/**/*'
+
+AddonManager:
+- 'src/Mod/AddonManager/**/*'
```

Comment:
Confused here, there is no `AddonManager` tag, so why reference it here?

Reply:
@luzpaz Yes, you are right.
That was an oversight on my part. The label would have been created if it matched --> I tested that and forgot to remove it.

@chennes I also changed the workflow trigger to always run the workflow, regardless of the workflow approval setting in Github. (See https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks ). I can make a pr for the codespell workflow if desired.